CoreTableView/src/au/gov/asd/tac/constellation/views/tableview2/TableViewPane.java

```diff
@@ -124,6 +125,9 @@
     private static final ImageView COPY_ICON = new ImageView(UserInterfaceIconProvider.COPY.buildImage(16));
     private static final ImageView EXPORT_ICON = new ImageView(UserInterfaceIconProvider.UPLOAD.buildImage(16));
     private static final ImageView SETTINGS_ICON = new ImageView(UserInterfaceIconProvider.SETTINGS.buildImage(16));
+    private static final ImageView MENU_ICON_SOURCE = new ImageView(UserInterfaceIconProvider.MENU.buildImage(16));
+    private static final ImageView MENU_ICON_DESTINATION = new ImageView(UserInterfaceIconProvider.MENU.buildImage(16));
+    private static final ImageView MENU_ICON_TRANSACTION = new ImageView(UserInterfaceIconProvider.MENU.buildImage(16));
```

Comment:
I'm not entirely familiar with ImageViews so feel free to correct me I'm wrong. Given the context that these 3 seem to be used, you may only need to create one instance of this ImageView which can be used for all of 3 rather than creating one for each

Reply:
I had the same idea, but for some reason I cannot reuse the same ImageView. The image was appearing only on the last place I used it.

Reply:
![image](https://user-images.githubusercontent.com/64826928/82277753-d50a8700-99c7-11ea-9007-5d4fd90754b0.png)
(Same when I tried to reuse the same filter on 3 sub menus- it works only on the last place)

Reply:
Yep that makes sense